Mission: Trist S&D - Cant Go Under It

TristGit opened the pull request:
Mission Pull Request Template

This Pull Request will:

Add the mission Can't Go Under It made by Trist to the repository.

**Mission Creation Checklist:**
- [x] Mission File
- [x] Ingame Play Area Markers
- [x] Ingame Briefings to all players
- [x] Side-based briefings
- [x] Objective Entities
- [x] Mission .conf file

**Faction(s):**
BLUFOR - Attacking/Defending/NA
- [ ] US
- [ ] UK
- [x] RHS_USAF 
- [ ] CDF

OPFOR - Attacking/Defending/NA
- [ ] USSR
- [x] RHS_AFRF

INDFOR - Attacking/Defending/NA
- [ ] FIA
- [ ] RHS_ION
- [ ] TALIBAN

**Objectives/Win Conditions:**

S&D

**Terrain:**

Chernarus

**Short mission description:**

A Russian convoy has been partially destroyed/immobilized around Krona castle from a targeted air strike. Both forces deploy mechanized platoons in order to maintain or destroy the decoupled cargo assets.

**Slotting Ratio:**

<3:2>

**Time limit:**

<60min>

**Mission Briefing Screenshot:**
![991e5f1b62908c6b928c5319e7267323](https://github.com/user-attachments/assets/15cda6a0-f906-411b-8eb0-b1fd2e4ec4eb)


**Design concept/thoughts:**

I had been wanting a mission where both objectives were potentially supportive of the other while maintaining a certain level of distinction between the two. The intent behind this mission being the presence of terrain that made several avenues of planning/approach with extreme asymmetry. The castle of Krona being an asset itself to whoever holds it, while asset and squad positioning helping it be a little more replayable yet important. 

Both teams can choose how aggressively they position themselves theoretically, while the sites rest in slight inopportune locations due to simulating a convoy that has been hit and hastily decoupling it's cargo. 

Bomb Site A lies within a small collection of buildings that overlays open fieldss with various disadvantageous/advantageous slopes around it.
Bomb Site B lies just outside the walls of Castle Krona, its hauling vehicle in a destroyed state. 

**Other Notes:**

- Both sides possess x2 of respective armed transports for additional support included with other transports. BLUFOR has x2 Unrmd Strykers for heavier transport.
- OPFOR contains 1x MMG Team, 1x Sniper (4 man, 1 marksman) Team
- BLUFOR contains 1x MMG Team
- Moderate building prefab amount compared to other missions to break/add sight lines, please notify if excessive
- NOT tested in server environment

The intention of the asset/unit comps is that OPFOR has its BTRs, Sniper Team, and MMG team to really lockdown areas of fire and punish exposure. While Blufor has a few angles of mass fire being laid down, asset allocation on Opfor's part can greatly limit objective approach. 

Had some help via feedback that there simply is no reason for Opfor to hold anything else if one of the sites were within a castle, so I used the idea of a hasty QRF-like response and placement of assets both sides and slightly inconvenient bomb site locations to simulate a hasty defense as a setting.

My first published mission so let me know of any technical, methodical, or general glaring issues or opinions

tanaaKa commented:
Adjust the slotting order so it's:

- Infantry platoons
- Infantry specialities
- Vehicles crews

tanaaKa commented:
Other feedback:

- Infantry platoons only go up to 5 squads at max due to both callsigns as well as overwhelming control elements
  - 6 means commander, 7 means 2IC or senior enlisted (PL and PSG)

- If you want there to be this many blufor capable, you will need to introduce a company element and two platoon elements.

- Reduce the blufor safe start area. They can move quite far
![image](https://github.com/user-attachments/assets/b9239440-4754-4d22-a4fb-610623accc56)

- You give both sides quite a few options for transport which will make balancing harder. I would reduce the choices for them.

- Toy with some different times and weather presets. This forest looks sick at different lighting.

- Also I hope you're not misrepresenting the reach of the castle Krona. Defenders could almost entirely stay in castle and still control both sites. Food for thought.

TristGit commented:
> Other feedback: .....
> 


Adjusted all of the above, castle in the end itself was supposed to be an asset itself for whoever held it WITHOUT it solely holding both sites but opened up the playzone around it with the intent that it would be a risk to do so if surrounded. Moved site A to be within the village Bogatyrka south, where more play around the tree canopy is possible.

Intent was to not have a single hard point of defense in the castle but still implementing it in some way. Wanted to make both sides have to evaluate potential positions and fire lines more.
